client/my-sites/sidebar/sidebar.jsx

```diff
@@ -155,14 +156,10 @@ export class MySitesSidebar extends Component {
 	}
 
 	ads() {
-		var site = this.getSelectedSite(),
+		const site = this.getSelectedSite(),
 			adsLink = '/ads/earnings' + this.siteSuffix();
 
-		if ( ! site || ! site.options.wordads ) {
-			return null;
-		}
-
-		return (
+		return ( ( site && site.options.wordads && userCan( 'manage_options', site ) ) &&
```

Comment:
I think it was a little more readable the other way IMHO. The `booleanCheck && <component/>` trick is more useful when you're using it within a larger block of JSX, and then it's best when used with a single boolean variable named something specific to the check like `canManageAds`.